ExampleCode/C++/API-Examples/README.md

```diff
@@ -0,0 +1,11 @@
+API Method Examples
+===================
+
+These cpp files are examples of the various methods of the iSENSE C++ Class.
+
+Use the provided Makefile to compile them, try them out and view the source code.
+
+Note - you will need to use a Unix like OS (Linux or Mac OS), have installed cURL and libcurl
+(installed by default by Mac OS!) and also have run the install script which pulls down the
```

Comment:
The script will only work for Linux, possibly Mac - I'll have to add a test case for Macs since there's no sudo apt-get. But it should in theory work on Macs. Windows works, and I have examples of making a Visual Studio project - but the script isn't necessary really since everything is provided in the example project. I may have to update those or make another script for people to easily get the API code (updated version) and examples / apps.